pages/_app.js

```diff
@@ -71,20 +78,29 @@ export default function App({
     }
   }
 
+  // Get words for user only
+  useEffect(() => {
+    if (databaseData) {
+      setWordList(
+        databaseData.filter((entry) => entry.userId[0] === activeUser)
+      );
```

Comment:
this step should be processed in the backend, since you don't want to send all the data from all users to the frontend only for it to be filtered there. You can hand the activeUser to your backend and only send the filtered wordlist back to your frontend.

Reply:
Yes, that makes a lot more sense actually. Do we actually have some documentation on that? I would have to send the userId via the API (GET request), wouldn't I? I'm currently just using SWR. 

Any hints? :)

Reply:
I created a call that sends the userId to the backend and only retrieves data that belongs to the user.